content/articles/image-adjustment-to-higher-or-lower-resolution-using-python/index.md

```diff
@@ -0,0 +1,238 @@
+
+The construction is like a pyramid with the lower quality image being upscaled till the end. Parameter sharing is frequently used for speed. It has two parts, the extraction stage, and the reconstruction stage. 
+
+The Laplacian pyramid is formed by the difference between the level in the `Gaussian Pyramid` and the extended version of its upper level in the pyramid. 
+
+Let us get started!
+
+### Instance 1
+### Scaling down the images to lower resolution
+We will scale down the images to lower resolutions in the Gaussian Pyramid model. To perform this task we will be using the image below:
+
+![sean](/engineering-education/image-adjustment-to-higher-or-lower-resolution-using-python/sean.jpg)
+
+
+```py
+import cv2
+import numpy as np
+img = cv2.imread("sean.jpg")#Here we will be reading the image
+layer = img.copy()
+gp = [layer] #Declaring a variable hence creating the Gaussian Pyramid array.
+for j in range(10):#Providing room for iteration.
+    layer = cv2.pyrDown(layer)
+    gp.append(layer)
+    cv2.imshow(str(j), layer)#This function displays the multiple images created.
+cv2.imshow("Original image", img)#This will have the original image displayed.
+cv2.waitKey(0)
+cv2.destroyAllWindows()
+```
+
+Your output should match the below image:
+
+![low](/engineering-education/image-adjustment-to-higher-or-lower-resolution-using-python/low.jpg)
+
+### Instance 2
+### Scaling up the images to higher resolutions.
+Once you increase the resolution of an already lowered resolution image, it will not go back to its original form. This is because lowering the resolution loses some information about the image using the `pyrDown` method hence the results look a little blurred. 
+
+> This is different from scaling up an original image.
+
+```py
+import cv2
+import numpy as np
+img = cv2.imread("sean.jpg")
+#We will be factoring down images using the already scaled.
+lwr1 = cv2.pyrDown(img)
+lwr2 = cv2.pyrDown(lwr1)
+lwr3 = cv2.pyrDown(lwr2)
+lwr4 = cv2.pyrDown(lwr3)
+# We will be Increasing the resolution of already scaled down image that is lwr4.
+hir1 = cv2.pyrUp(lwr3)
+
+cv2.imshow("Original image", img)
+cv2.imshow("First Scaled Down Image", lwr1)
+cv2.imshow("Second Scaled Down Image", lwr2)
+cv2.imshow("Third Scaled Down Image", lwr3)
+cv2.imshow("Fourth Scaled Down Image", lwr4)
+cv2.imshow("First Scaled Up Image", hir1)
+
+cv2.waitKey(0)
+cv2.destroyAllWindows()
+```
+
+The output should be similar to the below image:
+
+![high](/engineering-education/image-adjustment-to-higher-or-lower-resolution-using-python/high.jpg)
+
+### Instance 3
+To construct the Laplacian Pyramid Super-Resolution Network, we will be taking the top-level layer of the Gaussian Pyramid which is the last image generated using the loop function then create a list for the Laplacian pyramid then create the extended version of the upper level of the Gaussian Pyramid.
```

Comment:
This sentence is a whole paragraph, break it down and make sure it is comprehensive